<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="gameTabs">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Меню</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="mainPages">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="menuPage">
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="minimumSize">
                 <size>
                  <width>141</width>
                  <height>0</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <item>
                  <widget class="QPushButton" name="play">
                   <property name="text">
                    <string>Играть!</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="load">
                   <property name="text">
                    <string>Загрузить</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_17">
                   <property name="topMargin">
                    <number>11</number>
                   </property>
                   <item>
                    <widget class="QPushButton" name="save">
                     <property name="text">
                      <string>Сохранить</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QPushButton" name="autor">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="text">
                    <string>Авторы</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="settings">
                   <property name="text">
                    <string>Настройки</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="exit">
                   <property name="text">
                    <string>Выход</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QCheckBox" name="showIDE">
                   <property name="text">
                    <string>IDE</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="startGamePage">
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="startFFALayout"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Игра</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_23">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="layGl">
                <property name="spacing">
                 <number>0</number>
                </property>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Справка</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="manLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
